### PR TITLE
ENG-8511 - Update X & Y value Extraction from Touches

### PR DESCRIPTION
**Pull request by kevinsites**

### CODE REVIEW

**Comment by kishisaka-nid** [comment]
Looks good: 

(NeuroID Debug) Event:  TOUCH_START - 1724786243785 - Password - xy=[(x=95.33332824707031, y=350.0, tid=0)] tg=tgs: Password, location: UIControlSwizzle, sender: UITextField, etn: UITextField
(NeuroID Debug) NeuroID IH:  Adding NeuroID Health Event CUSTOM_TOUCH_END
(NeuroID Debug) Event:  CUSTOM_TOUCH_END - 1724786243897 - Password - xy=[(x=95.33332824707031, y=350.0, tid=0)] tg=etn: UITextField, location: gestureRecognizer, tgs: Password attrs=[size=0.0, force=0.0]

(NeuroID Debug) Event:  TOUCH_START - 1724786195647 - Email - xy=[(x=126.0, y=257.0, tid=0)] tg=location: UIControlSwizzle, etn: UITextField, tgs: Email, sender: UITextField
(NeuroID Debug) NeuroID IH:  Adding NeuroID Health Event CUSTOM_TOUCH_END
(NeuroID Debug) Event:  CUSTOM_TOUCH_END - 1724786195775 - Email - xy=[(x=126.0, y=257.0, tid=0)] tg=tgs: Email, etn: UITextField, location: gestureRecognizer attrs=[force=0.0, size=0.0]

These X/Y coords look like absolute touches against the entire window (login screen). Are these density independent pixel readings (similar to Android),. It looks like it since these are floats.

**Review by kishisaka-nid — APPROVED** [review]

File: NeuroID/TrackerClassExtensions/UIClassExtensions/UITextField.swift [hunk 1]
```diff
@@ -107,6 +98,18 @@ internal extension UITextField {
         textFieldSwizzling(element: textField,
                            originalSelector: #selector(textField.paste(_:)),
                            swizzledSelector: #selector(textField.neuroIDPaste))
+
+//        textFieldSwizzling(element: textField,
```
Comment (cometman) [inline NeuroID/TrackerClassExtensions/UIClassExtensions/UITextField.swift:102]: Do we still need this commented code block? When does neuroIDTouchStart/EndMove get called now? I set a breakpoint and didnt see `neuroIDTouchStart` get hit. I see the `CustomTapGestureRecognizer` which takes over this now Im assuming

**Review by cometman — APPROVED** [review]

**Review by priya-xavier — APPROVED** [review]
LGTM! I can see the xy coords in TOUCH events for text fields,UIButton, slider, stepper.

**Comment by kevinsites** [comment]
Bypassing Sonar because the duplicate code is class specific and can't be generalized

**Pull request merged**